.github/ISSUE_TEMPLATE/other-issues.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Does it bring any value to add an empty template? I newer worked with templates, so my question might look stupid 😅

Reply:
I haven't used them either, so I guess we'll learn together. 😄 From the screenshot in the [GH documentation](https://help.github.com/articles/about-issue-and-pull-request-templates/) there is a fine print "Open a regular issue", but I thought it might be easier for people if they see this as a main option as well.